test/examples/basic-open-api.json

```diff
@@ -24,6 +24,19 @@
                 ],
                 "title": "SuccessMessage",
                 "type": "object"
+            },
+            "WrappedInteger": {
+                "nullable": true,
```

Comment:
I think don't want this schema to be marked nullable in the components section, right? Isn't all the nullability handled by the inline schema that's nullable?

Reply:
Oops, thank you for noticing. I have addressed this in 537fe7738cd3f71f847957e1e2ba13a31604f5de